bindings/rascal/utils/scorer.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from scipy.stats import spearmanr
+
+def get_r2(y_pred,y_true):
```

Comment:
I feel this is really a copy/paste that can be avoided in most applications, if someone wants to have one of your suggestions then it can be implemented by the user.